src/input/command.zig

```diff
@@ -161,6 +162,19 @@ fn actionCommands(action: Action.Key) []const Command {
             .description = "Reset the font size to the default.",
         }},
 
+        .set_font_size => comptime &.{.{
+            .action = .{
+                .set_font_size = switch (builtin.os.tag) {
+                    // On macOS we default a little bigger since this tends to look better. This
+                    // is purely subjective but this is easy to modify.
+                    .macos => 13,
+                    else => 12,
+                },
+            },
+            .title = "Set Font Size",
+            .description = "Reset the font size to the default.",
+        }},
```

Comment:
We already have a reset font size command so we don't need this. I'll remove it.